arrow/arrow.py

```diff
@@ -722,7 +722,14 @@ def ambiguous(self):
 
         return dateutil_tz.datetime_ambiguous(self._datetime)
 
-    # mutation and duplication
+    @property
+    def imaginary(self):
+        """Returns a boolean indicating whether the :class: `Arrow <arrow.arrow.Arrow>` object exists in the current
+        timezone"""
```

Comment:
I'd compress this onto one line and add a period: Indicates whether the :class: `Arrow <arrow.arrow.Arrow>` object exists in the current timezone.